### PR TITLE
feat(cost-analysis-data-table): add usage_quantity_sum if usage type is included in groupby

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

image

### CODE REVIEW

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue [hunk 1]
```diff
@@ -258,6 +282,16 @@ const listCostAnalysisTableData = async (): Promise<CostAnalyzeResponse<CostAnal
...
 };
 
+const getUsageQuantity = (item: CostAnalyzeRawData, fieldName: string): number|string => {
+    const dateIndex = Number(fieldName.split('.')[1]);
+    const usageQuantity = item.usage_quantity_sum?.[dateIndex]?.value;
+    if (!usageQuantity) return 'unknown';
+    if (item.usage_unit === 'Bytes') {
+        return `${byteFormatter(usageQuantity, { unit: item.usage_unit })}`;
+    }
+    return usageQuantity.toFixed(2);
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue:292]: suggestion: better to use `numberFormatter()`!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue [hunk 2]
```diff
@@ -77,6 +85,8 @@ const state = reactive({
         [GROUP_BY.SERVICE_ACCOUNT]: state.serviceAccounts,
     })),
     visibleExcelNotiModal: false,
+    isIncludedUsageTypeInGroupBy: computed<boolean>(() => costAnalysisPageState.groupBy.includes('usage_type')),
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue:88]: nitpick: !!

```suggestion
    isIncludedUsageTypeInGroupBy: computed<boolean>(() => costAnalysisPageState.groupBy.includes(GROUP_BY.USAGE_TYPE)),
```

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue [hunk 3]
```diff
@@ -212,14 +253,20 @@ const listCostAnalysisTableData = async (): Promise<CostAnalyzeResponse<CostAnal
             data_source_id: costAnalysisPageStore.selectedDataSourceId,
             query: {
                 granularity: costAnalysisPageState.granularity,
-                group_by: costAnalysisPageState.groupBy,
+                group_by: [...costAnalysisPageState.groupBy, 'usage_unit'],
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue:256]: nitpick: It seems we don't need `usage_unit` when there's no `usage_type` in groupBy.

**Review by WANZARGEN — APPROVED** [review]
LGTM~

**Pull request merged**